packages/victory-candlestick/src/helper-methods.js

```diff
@@ -102,6 +102,32 @@ const getStyles = (props, style, defaultStyles = {}) => {
   };
 };
 
+// This method will edit or remove candlestick data points that fall outside of the desired domain
+// eslint-disable-next-line complexity
+const formatDataFromDomain = (datum, domainY) => {
+  const [minDomainY, maxDomainY] = domainY;
+  let { _low, _open, _close, _high } = datum;
+
+  // if single values fall outside of domain, set them to min or max
```

Comment:
I don't think we want to be altering single data values to match domain min / max since it could become misleading. Please only null out points when all values are outside of the desired domain

Reply:
understood, done.